tiledb/cf/engines/netcdf4_engine.py

```diff
@@ -601,15 +601,16 @@ def from_group(
             default_group_path=default_group_path,
         )
 
-    @classmethod
-    def _from_group_to_attr_per_array(
+    @classmethod  # noqa: C901
+    def _from_group_to_attr_per_array(  # noqa: C901
```

Comment:
Is this still needed after extracting `get_variable_chunks`?

Reply:
Yeah, it does.